src/coreclr/src/jit/morph.cpp

```diff
@@ -17624,22 +17628,90 @@ class LocalAddressVisitor final : public GenTreeVisitor<LocalAddressVisitor>
             return m_offset;
         }
 
+        // Get the location's field sequence.
+        FieldSeqNode* FieldSeq() const
+        {
+            return m_fieldSeq;
+        }
+
+        //------------------------------------------------------------------------
+        // Location: Produce a location value.
+        //
+        // Arguments:
+        //    lclVar - a GT_LCL_VAR node that defines the location
+        //
+        // Notes:
+        //   - (lclnum) => LOCATION(lclNum, 0)
+        //
+        void Location(GenTreeLclVar* lclVar)
+        {
+            assert(lclVar->OperIs(GT_LCL_VAR));
+            assert(!IsLocation() && !IsAddress());
+
+            m_lclNum = lclVar->GetLclNum();
+
+            assert(m_offset == 0);
+            assert(m_fieldSeq == nullptr);
+        }
+
+        //------------------------------------------------------------------------
+        // Location: Produce an address value from a GT_LCL_VAR_ADDR node.
```

Comment:
Location -> Address.

Reply:
Hah, I'll fix it with the next PR in this area.